src/App.js

```diff
@@ -42,9 +87,34 @@ class App extends React.Component {
           {this.state.guessedLetters.length > 0
             ? this.state.guessedLetters.toString()
             : "-"}
-          <h3>Input</h3>
+          <h4>You got {this.state.numOfGuessesLeft} guesses left.</h4>
+          <h5>Type in one letter into the input field.</h5>
           {/* Insert form element here */}
-          Todo: Insert form element here
+          <form onSubmit={this.handleSubmit}>
+            <input
+              type="text"
+              maxLength={1}
+              minLength={1}
+              name="input"
+              value={this.state.input}
+              onChange={this.handleChange}
+            />
+            <br />
+            <input type="submit" value={"Submit"} />
+          </form>
```

Comment:
```suggestion
          <form onSubmit={this.handleSubmit}>
            <input
              type="text"
              maxLength={1}
              minLength={1}
              name="input"
              value={this.state.input}
              onChange={this.handleChange}
            />
            <br />
            <input type="submit" value={"Submit"} />
          </form>
```

I think you could try without using a `<form>` tag, and rather use the submit button with an onClick instead. Basically fully controlling values and submission behaviour yourself, instead of relying on the form/submit-input html mechanics.